modules/instance_template/metadata.yaml

```diff
@@ -146,6 +148,10 @@ spec:
         description: Enable IP forwarding, for NAT instances for example
         varType: string
         defaultValue: "false"
+      - name: confidential_instance_type
+        description: Defines the confidential computing technology the instance uses. If this is set to "SEV_SNP", var.min_cpu_platform will be automatically set to "AMD Milan".
```

Comment:
Plz reference where to find the valid ENUMs?  Currently, it seems to just be for a single string, but CC will support other cpu platforms (i.e intel). I think the rpc might define a list, so we can mimic that somehow.

Reply:
feel free to resolve as won't fix, since we'll need to make some updates separately in a future PR.  It looks like we don't have a good definition of the enums, so for now, please feel free to resolve.